src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
+        section_def=Geometry,
+    )
+
+class SubstrateHolderPosition(ArchiveSection):
+    """
+    One casing position of the substrate holder.
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this position.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    slot_geometry = SubSection(
+        section_def=Geometry,
+    )
+    insert_reduction = SubSection(
+        section_def=InsertReduction,
+    )
+
+
+class SubstrateHolder(ArchiveSection):
+    """
+    The holder for the substrate.
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this position.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    lab_id = Quantity(
+        type=str,
+        description="""
+        The lab ID of the substrate holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
+        type=str,
+        description="""
+        The material of the holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    thickness = Quantity(
+        type=float,
+        unit='meter',
+        description="""
+        The thickness of the holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+            defaultDisplayUnit='micrometer',
+            ),
+    )
+    outer_diameter = Quantity(
+        type=float,
+        unit='meter',
+        description="""
+        The outer diameter of the substrate holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+            defaultDisplayUnit='millimeter',
+        ),
+    )
+    number_of_positions = Quantity(
+        type=int,
+        description="""
+        The number of positions on the holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+        ),
+    )
+    image = Quantity(
+        type=str,
+        description="""An image of the substrate holder.""",
+        a_browser={'adaptor': 'RawFileAdaptor'},
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.FileEditQuantity,
+        ),
+    )
+    positions = SubSection(
+        section_def=SubstrateHolderPosition,
+        repeats=True,
+    )
+
+
+class FilledSubstrateHolderPosition(SubstrateHolderPosition):
+    """
+    One casing position of the filled substrate holder.
+    """
+    substrate = SubSection(
+        section_def=CompositeSystemReference,
+        description="""
+        The substrate that is placed in this position.
+        """,
+    )
+class FilledSubstrateHolder(SubstrateHolder):
```

Comment:
Is this the class for Oliver or is this one that you think we can nest in the process? Wondering if we should inherit `Entity`/`Instrument` or not.

Reply:
This can be used in both cases
I can make them entity, yes. I will